cli/src/Morphir/Web/DevelopApp.elm

```diff
@@ -2135,7 +2135,7 @@ viewDefinitionDetails model =
         viewRawValue config ir rawValue =
             case fromRawValue ir rawValue of
                 Ok typedValue ->
-                    el [ centerY ] (ViewValue.viewValue config typedValue)
+                    el [ centerY ] (ViewValue.viewValue 1000 config typedValue)
```

Comment:
It would be good to move the specific numbers out to named variables and also add some explanation on why the particular value was chosen. I see 1000 and 9999 but not sure why those are the values.